code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+	route = "Flesh"
+	var/max_amt = 2
+	var/current_amt = 0
+	var/list/ghouls = list()
+
+/datum/eldritch_knowledge/flesh_ghoul/on_finished_recipe(mob/living/user,list/atoms,loc)
+	var/mob/living/carbon/human/H
+	for(var/mob/living/carbon/human/humie in atoms)
+		H = humie
+	if(!H)
+		return
+	var/mob/dead/observer/G
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
+		if(ghost.mind && ghost.mind.current == H && ghost.client)  //the dead mobs list can contain clientless mobs
+			G = ghost
+			break
+	if(!G.reenter_corpse())
+		var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [H.real_name], a voiceless dead", ROLE_ECULTIST, null, ROLE_ECULTIST, 50,H)
+		if(!LAZYLEN(candidates))
+			return
+		var/mob/dead/observer/C = pick(candidates)
+		message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(H)]) to replace an AFK player.")
+		H.ghostize(0)
+		H.key = C.key
+	if(!H.mind || !H.client)
+		to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+		return
+	if(!check_ghouls() || HAS_TRAIT(H,TRAIT_GHOUL))
+		return
+
+	ADD_TRAIT(H,TRAIT_MUTE,MAGIC_TRAIT)
+	ADD_TRAIT(H,TRAIT_GHOUL,MAGIC_TRAIT)
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(50)
+	H.health = 50 // Voiceless dead are much tougher than ghouls
+	H.fully_replace_character_name(H.real_name,"Voiceless [H.real_name]")
+	to_chat(H, "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>")
+	to_chat(H, "<span class='userdanger'>[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!</span>")
+	atoms -= H
+	ghouls += H
+
+/datum/eldritch_knowledge/flesh_ghoul/proc/check_ghouls()
+
+	listclearnulls(ghouls)
+
+	for(var/mob/living/carbon/human/ghoul in ghouls)
+		if(ghoul.stat == DEAD)
+			ghouls -= ghoul
+			current_amt--
+
+	if(current_amt >= max_amt)
+		return FALSE
+	return TRUE
+
+/datum/eldritch_knowledge/flesh_grasp
+	name = "Grasp of Flesh"
+	gain_text = "My new found desire, it drove me to do great things! The Priest said."
+	desc = "Empowers your mansus grasp to be able to create a single ghoul out of a dead person. Ghouls have only 25 HP and look like husks."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_ghoul)
+	var/ghoul_amt = 1
+	var/list/spooky_scaries
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_grasp/mansus_grasp_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!ishumanbasic(target))
+		return
+	var/mob/living/carbon/human/H = target
+	var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)
+	if(E)
+		E.on_effect()
+		H.bleed_rate = min(H.bleed_rate + 4, 8)
+	if(H.stat != DEAD)
+		to_chat(user, "<span class='warning'>This spell can only affect the dead!</span>")
```

Comment:
but what if you're just using mansus grasp on a living person to deal 10 brute damage, cause a second of knockdown, or  proc the laughably bad flesh mark effect?

Reply:
i will have to update flesh mark, oh and another good catch